analysis/test/merge

```diff
@@ -7,17 +7,17 @@ import sys, os
 # ///      otherwise only specific dir name where root files are in       ///
 # ///////////////////////////////////////////////////////////////////////////
 
-user_input = sys.argv[1]
+#user_input = sys.argv[1]
 """
 if "results_merged" not in os.listdir(os.getcwd()):
     os.system("mkdir results_merged")
     print "'results_merged' has been created."
 """
 
 version="%s"%os.environ["CMSSW_VERSION"]
-dirname = "test"
-pstored="/xrootd/store/user/%s/nanoAOD/%s/"%(os.environ['USER'],user_input)
-#pstored = "/cms/scratch/daniel/nanoAOD/src/nano/analysis/h2mu/Results/CSV/"
+#dirname = "180703"
+#pstored="/xrootd/store/user/%s/nanoAOD/test/%s/"%(os.environ['USER'],user_input)
+pstored="/xrootd_user/seulgi/xrootd/nanoAOD/test181130_2/"
```

Comment:
try to have this generic so all users can use it

Reply:
Will this change affect other users then? It shouldn't be committed if so